src/YoloObjectDetector.cpp

```diff
@@ -136,11 +154,30 @@ void YoloObjectDetector::on_image_callback(const cv::Mat& image)
     frameHeight_ = image.rows;
     if (!writer_configured_)
     {
-      std::string url_writer(this->get_namespace());
-      url_writer = "rtsp://127.0.0.1:8554"+url_writer;
-      std::cout << "url_writer: " << url_writer << std::endl;
-      // Configure RTSP Streamer
-      rtsp_streamer_.on_configure_writer(frameWidth_, frameHeight_, 30, 9000, url_writer);
+      // Check whether a GStreamer pipeline has been provided
+      std::string gstreamer_pipeline;
```

Comment:
We also need to be able to configure the reader pipeline. We already discussed about it but I cannot see the changes.

Reply:
@vicmassy I left a comment on another `umd_darknet` PR; the main point is that `umd_rtsp` does not offer an `on_configure_reader` method that accepts a custom pipeline as an input. https://github.com/umdlife/darknet_ros/pull/2#discussion_r1021202706

If we want to do that, we would need to open a new PR on `umd_rtsp` to implement that, then come back to this point.